docs/platform/concepts/orgs-units-projects.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Organizations, units, and projects
+---
+
+The Aiven platform uses organizations, organizational units, and projects to organize services and manage access.
+
+import Overview from "@site/static/images/content/figma/organizations-overview.png";
+
+<img alt="Hierarchy showing two organizational units, each with two projects, nested within one organization." src={Overview} width="58%" class="centered"/>
+
+## Organizations and organizational units
+
+Organizations and organizational units are collections of projects. When
```

Comment:
I would rephrase this sentence a bit something along these lines.

An Organization enables you to run and manage Aiven services efficiently and secure. Organizational units are collection of projects and can be used to group projects together.

Maybe we can work with Janki to come up with a better wording.

Reply:
I added a little of this to the intro, but let's keep it focused more on the functionality and less on the marketing-y language. :)